### PR TITLE
Add API CR and fix Application CR name bug

### PR DESCRIPTION
**Pull request by colunira**

### CODE REVIEW

File: chart/githubconnector/README.md [hunk 1]
```diff
@@ -20,6 +20,14 @@ helm install {HELM_CHART_DIRECTORY} --tls
 ```
 **NOTE:** To define namespace in which chart should be installed add flag `--namespace`. You can also define name of your release with flag `--name`.
 
+**ATTENTION:** Our application is in Beta version. For now you HAVE TO specify those flags:
+* --name - it has to be "github-connector"
+* --set conatiner.image={VALUE} - specify it if you have newer version of docker image than karoljaksik/github-connector:1.0.1
```
Comment (KarolJaksik) [inline chart/githubconnector/README.md]: Small typo, should have been
`set container.image={VALUE} `

Comment (KarolJaksik) [inline chart/githubconnector/README.md]:
```suggestion
* --set container.image={VALUE} - specify it if you have newer version of docker image than karoljaksik/github-connector:1.0.2
```

File: chart/githubconnector/README.md [hunk 2]
```diff
@@ -20,6 +20,14 @@ helm install {HELM_CHART_DIRECTORY} --tls
...
+* --set kymaAddress={VALUE} - specify your kyma adddress (for example 35.195.198.66.xip.io)
+```
+helm install --set container.image=karoljaksik/github-connector:1.0.1 --set kymaAddress=35.195.198.66.xip.io -n github-connector --namespace flying-seals . --tls
```
Comment (rafalpotempa) [inline chart/githubconnector/README.md]:
```suggestion
helm install --set container.image=karoljaksik/github-connector:1.0.2 --set kymaAddress=35.195.198.66.xip.io -n github-connector --namespace flying-seals . --tls
```

File: chart/githubconnector/values.yaml [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+  image: karoljaksik/github-connector:1.0.2
   containerPort: 8080
   limits:
     memory: "128Mi"
     cpu: "500m"
 
 service:
   port: 80
+
+kymaAddress: 35.195.198.66.xip.io
```
Comment (rafalpotempa) [inline chart/githubconnector/values.yaml]: I think that in future this should be imported from environment.

Comment (colunira) [reply chart/githubconnector/values.yaml]: It's in team's backlog as a task for future

**Pull request merged**